xwayland/selection/incoming.c

```diff
@@ -28,7 +28,7 @@ static int xwm_data_source_write(int fd, uint32_t mask, void *data) {
 		xwm_selection_transfer_destroy_property_reply(transfer);
 		xwm_selection_transfer_remove_event_source(transfer);
 		xwm_selection_transfer_close_wl_client_fd(transfer);
-		return 1;
+		return 0;
```

Comment:
The docs say:

> If the event source is registered for re-check with wl_event_source_check(): 0 for all done, 1 for needing a re-check. If not registered, the return value is ignored and should be zero. 

The event source has not be registered with `wl_event_source_check`, so the return value is ignored.

I can see how this can be useful in the next commit though.

Reply:
Good point, I've fixed the commit message.